### PR TITLE
solution

### PR DESCRIPTION
**Pull request by VenomSRT**
- [DEMO LINK](https://VenomSRT.github.io/js_task_fix_form_DOM/)

### CODE REVIEW

**Review by yuriimovchko — APPROVED** [review]
Можна було розділити слова.... а так все круто

**Review by yuriimovchko — APPROVED** [review]

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const fieldContainers = document.querySelectorAll('.field');
+
+for (const field of [...fieldContainers]) {
+  const thisFieldInput = field.querySelector('input');
```
Comment (irenhh) [inline src/scripts/main.js:6]: why `thisFieldInput`?

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const fieldContainers = document.querySelectorAll('.field');
```
Comment (irenhh) [inline src/scripts/main.js:3]: fieldContainers?

Comment (irenhh) [reply src/scripts/main.js:3]: I have no idea what this could be, if I don't go to the demo link and search for it

**Review by irenhh — CHANGES_REQUESTED** [review]
separate words, use the reference picture

**Comment by irenhh** [comment]
deploy

**Comment by irenhh** [comment]
check your demo link, comments are not fixed on it

**Review by irenhh — APPROVED** [review]

**Pull request closed without merging**